packages/cli/src/config/bitrate.ts

```diff
@@ -17,3 +17,27 @@ export const setVideoBitrate = (bitrate: string | null) => {
 export const getVideoBitrate = () => {
 	return videoBitrate;
 };
+
+let maxRate: string | null = null;
+
+export const setVideoMaxRate = (bitrate: string | null) => {
+	maxRate = bitrate;
+}
+
+export const getVideoMaxRate = () => {
+	return maxRate;
+}
+
+/**
+ * bufSize is not a bitrate, but it is a bitrate-related option and get validated like a bitrate.
+ */
+let bufSize: string | null = null;
```

Comment:
Should I let bufSize setter and getter in the bitrate.ts for "simplicity" or should I create a new file ?